components/03-modules/picture/Picture.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <picture class="image ">
+    <source
+      v-for="(image, index) in images"
+      :key="'image_' + index"
+      :media="maxWidth(image['max-width'])"
```

Comment:
I think you can avoid using method, try
```
:media="image['max-width'] ? `(max-width: ${image['max-width']})` : false"
```
instead

Reply:
or you can:

`:media="image['max-width'] && (max-width: ${image['max-width']}) `